crates/shared/src/price_estimation/factory.rs

```diff
@@ -155,7 +155,7 @@ impl<'a> PriceEstimatorFactory<'a> {
             .as_ref()
             .and_then(|trade_verifier| estimator.verified(trade_verifier));
 
-        let fast = instrument(estimator, name);
+        let fast = instrument::<T>(estimator, name);
```

Comment:
is `::<T>` actually needed here? I thought the compiler would be smart enough to deduce `T` on it's own.